### ISSUE TITLE
LUA not blocking keys

### ISSUE THREAD

**Issue by mi-der** [TaranVH/2nd-keyboard#23]
***TL;DR at end***

hi! so, I copied your code exactly and only changed the keypress.txt file path.  My hotkeys do what they are supposed to, except LuaMacros isn't blocking my original keystroke
Example:
I work in visual studio a lot so i press "F" to add full screen code, except this is what shows up-
```
        fMe.FormBorderStyle = FormBorderStyle.None
        Me.WindowState = FormWindowState.Maximized
```
I actually hit the key to insert that exactly as it does.  I could just delete this, but Visual Studio auto-completes commands so this corrects to:
```
        OutOfMemoryException.FormBorderStyle = FormBorderStyle.None
        Me.WindowState = FormWindowState.Maximized
```
This is because it sees the "fMe" as in-OfMemory and corrects it.  I could "bodge" it and add {Up}{Left 39}{BS 20}, But this happens with every key, even unbound keys.  For example I could hit wertyuiopjz on my second keyboard (which i just did) and the key will still be inserted.  I am farily certain this isn't an AHK issue, because I didn't actually take any of taran's code and it is very simple-(http://pastebin.com/bH8C6jxv)

TL;DR
LuaMacros isn't blocking the keystroke from my second keyboard,  So Pressing my F macro will insert F aswell as performing the macro

**Comment by mi-der** [comment]
I am unsure of what the issue was, but it appears to be working now.

**Comment by mi-der** [comment]
Okay, I found out what had happened.  Last night, i rearranged some keys and fixed the code.  Today, I added more code.  For some reason, When i launch the script(Right Click>Run) Everything acts as it did last night (Ex: The F key does nothing).  Now, when i right click on it in the taskbar and click reload, what i described above happens.  I have no idea how or why this happens, but i will try to investigate.

**Comment by mi-der** [comment]
I Got It!  I have no idea what it was,but i just copied the code to a new file and everything worked!  I assume it was some kind of idiotic windows bug, but I'm glad i sorted this out and hopefully i helped someone
